docs/source/isaura.rst

```diff
@@ -1,5 +1,163 @@
+
+
+ - It is possible that some spurious **low-energy** hits appear around the track (due to over-iterations during the Richarson-Lucy deconvolution process, as commented in :doc:`beersheba`; or some noise inside the chamber, for example). If these hits are reconstructed around the track but not far enough to produce a different S2 or track (taking into account the voxel size), they can be considered as a part of the main one and, being a bit separate, it is probable that they end up belonging to an extreme voxel. That case would not be correct, and in order to solve it, the voxel will be dropped from the track and its energy passed to the closest one. This process is only carried out if the voxel energy is lower than: ``energy_threshold`` and the track is made by more than ``min_voxels`` voxels. Once this procedure is done, the extreme voxels are searched and found again recursively, until none of these conditions are fulfilled.
+
+ - Another particular scenario is the one that comes up when there are multiple end-voxel candidates (one can imagine that the shorter the track the more probable is this to happen). To deal with it, the more energetic candidates will be the ones set as extremes. With this convention, we aim to minimize the voxel-dropping algorithm commented just above.
+
+
+Eventually, once the extreme voxels are properly found, the computation of the center and energy of the blobs (stored in the ``Tracks/Tracking`` table as: ``blobi_x``, ``blobi_y``, ``blobi_z``, and ``eblobi`` (with ``i`` being 1, 2), respectively) will be rather straightforward, just as the previous figure pointed out. The only comment to take into account here is that not every hit falling inside the blob sphere will be considered for its total energy, but only the ones that belong to a voxel adjacent to the extreme.
+
+As convention, the assignment of ``1`` and ``2`` in defined in such a way that ``eblob1`` > ``eblob2``.
+
+
+The final step of the *Paolina* algorithm includes the computation of the ``ovlp_blob_energy`` (“*overlap blob energy*”) variable: in short tracks it is common to have **overlapping blobs**, i.e. blobs that share some of their hits [#]_. In these cases, the blob energies are not computed correctly, since they would produce an over-estimation of the total energy of the track. Therefore, it will be interesting to reject this type of events during the posterior analysis in case the blob energy distributions want to be studied. Owing to the fact that ``ovlp_blob_energy`` is defined as the total amount of energy of these shared hits, a selection of ``ovlp_blob_energy = 0`` will get rid of these events easily.
```

Comment:
` In these cases, the blob energies are not computed correctly, since they would produce an over-estimation of the total energy of the track.`: I'm not sure I understand the sentence correctly. One can calculate the blob energies not correctly if a hit belongs to both blobs, but the total energy of the track is still computed correctly, because it is not calculated from the blob energy, right?

Reply:
Yes, you are right. It is obvious that the energy of the blob does not affect the total energy of events, but what I meant is that if there is some overlapping, the total energy of the blobs makes no sense because it can be even larger than the total energy of the event. I'll rephrase it.